src/bundle/Resources/public/scss/_view-content.scss

```diff
@@ -1,6 +1,7 @@
 .ez-content-view {
     .ez-content-container {
         display: flex;
+        min-width: 0;
```

Comment:
Note: this is needed because we do not want the content (tables) to dictate container width.
The problem is described here: https://css-tricks.com/flexbox-truncated-text/#article-header-id-3
and exact `min-width` behaviour in flexbox was specified here: https://www.w3.org/TR/css-flexbox-1/#min-size-auto